spec/models/user_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+def test_wrong_user(params = {})
+  wrong_user = User.create(params)
+  expect(wrong_user.persisted?).to be_falsy
+end
+
+RSpec.describe User, type: :model do
+  let!(:email) { "user@example.com" }
+  let!(:password) { "password" }
+  let!(:first_name) { "louis" }
+  let!(:last_name) { "ramos" }
+  let!(:user) { User.create(email:, password:, first_name:, last_name:) }
```

Comment:
To create a test with RSpec I have to describe a new RSpec by specifying the name of the Class, in this case User, and its type, in this case User is a model.
Then I created a few "instance variables" (they're not called that but let's called them that, it's the same principle) to be able to access them in all of my tests without having to rewrite them all everytime